deploy/ocpbugs-15043/script.sh

```diff
@@ -0,0 +1,136 @@
+#!/bin/bash
+
+set -euo pipefail
+
+CHECKED_NAMESPACES=(
+    "openshift-sre-pruning"
+    "openshift-monitoring"
+)
+
+LAST_SL_CONFIGMAP="ocpbugs-15043-last-sl-sent"
+LAST_RUN_ANNOTATION="lasttimestamp"
+DELAY_BETWEEN_SEND_SECONDS="86400"
+
+NODES=()
+
+MACHINE_API_NAMESPACE="openshift-machine-api"
+
+for current_ns in "${CHECKED_NAMESPACES[@]}"; do
+    readarray -t failedPods < <(oc -n "$current_ns" get pods -owide | grep "CreateContainerError" | awk '{print $1}')
+    for pod in "${failedPods[@]}"; do
+        echo "Found pod $pod failing."
+        podDescribe="$(oc -n "$current_ns" describe pod "$pod")"
+
+        if ! echo "$podDescribe" | grep "error loading seccomp filter into kernel" >/dev/null; then
+            echo -e "\t$pod is failing for another reason. Ignoring"
+            continue
+        fi
+
+        node="$(echo "$podDescribe" | grep Node: | awk '{print $2}' | cut -d "/" -f 1)"
+        echo -e "\t$pod on $node having seccomp issues"
+        if echo "${NODES[@]}" | grep "$node" >/dev/null; then
+            continue
+        fi
+        NODES+=("$node")
+    done
+done
+
+if [ ${#NODES[@]} -eq 0 ]; then
+    echo "No affected nodes."
+    exit 0
+fi
+
+# print affected nodes and 
+
+for NODE in "${NODES[@]}"; do
+    nodeJson="$(oc get node "$NODE" -ojson)"
+
+    # make sure we only restart infra nodes
+    if ! jq -e '.metadata.labels."node-role.kubernetes.io/infra"' <<< "$nodeJson" > /dev/null ; then
```

Comment:
what is the rational for only sticking to infra nodes?

Reply:
Mostly this is here to be 100% sure we don't delete a control plane node accidentally and that we don't interfere with CU workloads. The pods that are monitored by this script should only be running on infra nodes anyway, so in practice, this might not even have any effect.